packages/jasmine/index.js

```diff
@@ -1,4 +1,14 @@
-const jasmineCore = require('jasmine-core');
+const path = require('path');
+
+// We want the jasmine-core version that is a transitive dependency of jasmine, however,
```

Comment:
I'm thinking, if we need `jasmine-core` here, should it be part of the dependencies of this package so we don't have to guess where it gets installed?

Reply:
Ahhh.. Yes. That would be a much cleaner solution. @bazel/jasmine would have a dependency on the same version of jasmine-core that its jasmine dep does. Good call 😄